audio_clip_encoder.py

```diff
@@ -52,14 +52,10 @@ def _get_input_data(self, docs: DocumentArray, parameters: dict):
         return filtered_docs
 
     def _create_embeddings(self, filtered_docs: Iterable):
-        """Update the documents with the embeddings generated by VGGISH"""
+        """Update the documents with the embeddings generated by AudioCLIP"""
 
         for d in filtered_docs:
-            # Vggish broadcasts across different length audios, not batches
-            audio = [d.blob, deepcopy(d.blob)]
-            audio = torch.stack([self.audio_transforms(track.reshape(1, -1)) for track in audio])
-            ((audio_features, _, _), _), _ = self.aclp(audio=audio)
-            audio_features = audio_features[0]
-            audio_features = audio_features / torch.linalg.norm(audio_features, dim=-1, keepdim=True)
-
-            d.embedding = audio_features.numpy()
+            track = d.blob
+            audio = torch.stack([self.audio_transforms(track.reshape(1, -1))])
```

Comment:
why is the reshape needed or the `stack` is it weird to use `stack` when a signle `blob` is used